packages/Panel/README.md

```diff
@@ -49,28 +97,32 @@ npm install @paprika/panel
 You can use any of the following components to compose the Panel:
 
 ```js
+<Panel.Content />
 <Panel.Header />
 <Panel.Footer />
 <Panel.Overlay />
 <Panel.FocusLock />
 <Panel.Trigger />
 ```
 
-## Basic examples.
+## Basic examples
 
 ```js
```

Comment:
nit: I think the syntax highlighting in the MDX page will look better with `jsx` instead of `js`